libs/core/src/lib/inline-help/inline-help.directive.ts

```diff
@@ -139,4 +132,19 @@ export class InlineHelpDirective extends BasePopoverClass implements OnInit, OnC
             this.additionalBodyClass += ' ' + INLINE_HELP_ICON_CLASS;
         }
     }
+
+    /** @hidden */
+    private _setupScreenreaderElement(content: string | Nullable<TemplateRef<any>>): void {
+        let srElement = document.createElement('span');
```

Comment:
Since this folk is called from the input and it is very likely that it will be very important for this component to be rendered before serving to the client (SSR use case), I think this should be modified a little bit:
- Component should have `Renderer2` injected and code will look like this `let srElement = this.renderer2.createElement('span');`
- Optionally you can append using the `renderer2.appendChild`